CarRental.Api.Host/Program.cs

```diff
@@ -0,0 +1,89 @@
+using AutoMapper;
+using CarRental.Api.Host;
+using CarRental.Application;
+using CarRental.Application.Contracts.CarModels;
+using CarRental.Application.Contracts.Cars;
+using CarRental.Application.Contracts.Clients;
+using CarRental.Application.Contracts.ModelGenerations;
+using CarRental.Application.Contracts.RentalCars;
+using CarRental.Application.Contracts.Reports;
+using CarRental.Application.Services;
+using CarRental.Domain.Interfaces;
+using CarRental.EFCore;
+using CarRental.EFCore.Repositories;
+using CarRental.ServiceDefaults;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.OpenApi.Models;
+using MySqlConnector;
+using System.Reflection;
+
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.AddServiceDefaults();
+
+builder.AddMySqlDataSource("CarRentalDb");
```

Comment:
После изменения пакета тут код будет выглядеть так:
```suggestion
builder.AddMySqlDbContext<CarRentalDbContext>("CarRentalDb");
```
Он автоматически создаст нужный DbContext и не нужно будет ниже парится с его созданием.